test/integration/channel-transfer-multi/channel_transfer_multi_test.go

```diff
@@ -0,0 +1,495 @@
+			Eventually(redisProcess.Wait(), time.Minute).Should(Receive())
+		}
+	})
+
+	BeforeEach(func() {
+		By("add admin to acl")
+		client.AddUser(network, peer, network.Orderers[0], admin)
+
+		By("add user to acl")
+		var err error
+		user, err = client.NewUserFoundation(pbfound.KeyType_ed25519)
+		Expect(err).NotTo(HaveOccurred())
+
+		client.AddUser(network, peer, network.Orderers[0], user)
+
+		By("emit tokens")
+		client.NBTxInvokeWithSign(network, peer, network.Orderers[0], nil, cmn.ChannelIndustrial, cmn.ChannelIndustrial,
+			admin, "initialize", "", client.NewNonceByTime().Get())
+
+		initialBalances = []*cligrpc.TransferItem{
+			{
+				Token:  "INDUSTRIAL_202009",
+				Amount: "10000000000000",
+			},
+			{
+				Token:  "INDUSTRIAL_202010",
+				Amount: "100000000000000",
+			},
+			{
+				Token:  "INDUSTRIAL_202011",
+				Amount: "200000000000000",
+			},
+			{
+				Token:  "INDUSTRIAL_202012",
+				Amount: "50000000000000",
+			},
+		}
+
+		transferItems = []*cligrpc.TransferItem{
+			{
+				Token:  "INDUSTRIAL_202009",
+				Amount: "1000000000000",
+			},
+			{
+				Token:  "INDUSTRIAL_202010",
+				Amount: "10000000000000",
+			},
+			{
+				Token:  "INDUSTRIAL_202011",
+				Amount: "20000000000000",
+			},
+			{
+				Token:  "INDUSTRIAL_202012",
+				Amount: "5000000000000",
+			},
+		}
+
+		expectedIndustrialBalances = []*cligrpc.TransferItem{
+			{
+				Token:  "INDUSTRIAL_202009",
+				Amount: "9000000000000",
+			},
+			{
+				Token:  "INDUSTRIAL_202010",
+				Amount: "90000000000000",
+			},
+			{
+				Token:  "INDUSTRIAL_202011",
+				Amount: "180000000000000",
+			},
+			{
+				Token:  "INDUSTRIAL_202012",
+				Amount: "45000000000000",
+			},
+		}
+
+		for _, initial := range initialBalances {
+			group := strings.Split(initial.GetToken(), "_")[1]
+
+			client.Query(network, peer, cmn.ChannelIndustrial, cmn.ChannelIndustrial,
+				fabricnetwork.CheckResult(fabricnetwork.CheckIndustrialBalance(group, initial.Amount), nil),
+				"industrialBalanceOf", admin.AddressBase58Check)
+
+			client.TxInvokeWithSign(network, peer, network.Orderers[0], cmn.ChannelIndustrial, cmn.ChannelIndustrial,
+				admin, "transferIndustrial", "", client.NewNonceByTime().Get(), nil,
+				user.AddressBase58Check, group, initial.Amount, "transfer industrial tokens")
+
+			client.Query(network, peer, cmn.ChannelIndustrial, cmn.ChannelIndustrial,
+				fabricnetwork.CheckResult(fabricnetwork.CheckIndustrialBalance(group, initial.Amount), nil),
+				"industrialBalanceOf", user.AddressBase58Check)
+		}
+	})
+
+	It("multi transfer by admin test", func() {
+		By("creating grpc connection")
+		clientCtx = metadata.NewOutgoingContext(context.Background(), metadata.Pairs("authorization", networkFound.ChannelTransfer.AccessToken))
+
+		transportCredentials := insecure.NewCredentials()
+		grpcAddress := networkFound.ChannelTransfer.HostAddress + ":" + strconv.FormatUint(uint64(networkFound.ChannelTransfer.Ports[cmn.GrpcPort]), 10)
```

Comment:
It seems that there are functions in standard libraries that make address from host and port